### PR TITLE
chore: Add Commit Checker github action

### PR DESCRIPTION
**Pull request by rimolive**
**Description of your changes:**

**Checklist:**
- [X] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by rimolive** [comment]
::group::Commit Checker results
::warning ::==== These are the results of the commit checker scans         ====
::warning ::==== If these are not commits from upstream kfp, then          ====
::warning ::==== please ensure you adhere to the commit checker formatting ====
::endgroup::

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

File: tools/commit_checker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+VOLUME ["/src/app-root"]
+
+LABEL name="Commit Checker tool" \
+      summary="commitchecker validates a range of commits in a git repository and ensures they meet specific requirements: \
+        1. The author's email address does not start with "root@". \
+        2. The message starts with one of: \
+            i. UPSTREAM: <PR number|carry|drop>: description \
+            ii. UPSTREAM: revert: \
+        This is useful for repositories that are downstream forks of upstream repositories."
```
Comment (HumairAK) [inline tools/commit_checker/Dockerfile:28]: can you add a newline

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Comment by dsp-developers** [comment]
Change to PR detected. A new PR build was completed.
  A set of new images have been built to help with testing out this PR: 
  **API Server**: `quay.io/opendatahub/ds-pipelines-api-server:pr-38`
  **DSP DRIVER**: `quay.io/opendatahub/ds-pipelines-driver:pr-38`
  **DSP LAUNCHER**: `quay.io/opendatahub/ds-pipelines-launcher:pr-38`
  **Persistence Agent**: `quay.io/opendatahub/ds-pipelines-persistenceagent:pr-38`
  **Scheduled Workflow Manager**: `quay.io/opendatahub/ds-pipelines-scheduledworkflow:pr-38`
  **MLMD Server**: `quay.io/opendatahub/mlmd-grpc-server:latest`
  **MLMD Envoy Proxy**: `registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.3.9-2`
  **UI**: `quay.io/opendatahub/ds-pipelines-frontend:pr-38`

**Pull request merged**